Kernel/Library/ThreadSafeRefPtr.h

```diff
@@ -49,7 +49,7 @@ struct RefPtrTraits {
 
     ALWAYS_INLINE static bool is_null(FlatPtr bits)
     {
-        return !(bits & ~(FlatPtr)1);
+        return 0 == (bits & ~(FlatPtr)1);
```

Comment:
Here and again in `lock()`: Why the Yoda condition?

Reply:
I won't reply to all these but the reason is personal style and what came naturally when fixing them 🤷‍♂️. Will make them consistent

Reply:
Sure, I just flagged them all in the hope of making it easier for you to find them.

Again, I actually prefer Yoda-conditions, I just think it's not the usual project style. So in the places that you disagree (like after "huge expressions" as you said), go ahead and do a Yoda comparison.